apps/chat-e2e/src/tests/sideBarFilters.test.ts

```diff
@@ -3,30 +3,36 @@ import { FolderInterface } from '@/chat/types/folder';
 import { Prompt } from '@/chat/types/prompt';
 import dialTest from '@/src/core/dialFixtures';
 import {
-  ExpectedConstants,
-  ExpectedMessages,
   FilterMenuOptions,
   FolderConversation,
   FolderPrompt,
-  ModelIds,
 } from '@/src/testData';
-import { expect } from '@playwright/test';
+import { GeneratorUtil, ModelsUtil } from '@/src/utils';
 
-dialTest.skip(
-  'Filter "Shared by me" shows only shared chats',
+const nestedLevel = 2;
+const searchTerm = 'test';
+
+dialTest(
+  'Filter "Shared by me" shows only shared chats.\n' +
+    'Filter "Shared by me" stays checked if to search chats',
   async ({
     dialHomePage,
     conversations,
     folderConversations,
+    mainUserShareApiHelper,
+    additionalUserShareApiHelper,
     conversationData,
     dataInjector,
     localStorageManager,
     chatFilter,
     chatBar,
     chatFilterDropdownMenu,
+    chatBarSearch,
+    conversationAssertion,
+    chatBarFolderAssertion,
     setTestIds,
   }) => {
-    setTestIds('EPMRTC-1597');
+    setTestIds('EPMRTC-1597', 'EPMRTC-1631');
```

Comment:
![image](https://github.com/user-attachments/assets/7e6c2389-c92b-4632-8147-bacf69f32a8e)
In the EPMRTC-1631 it is said to share just a conversation, not a convertsation in a folder. Do we consider that a correct substitute?

Reply:
I guess if the filtering works for chats within folders, it will work for single chats as well.  However, to confirm this, let me add one more single conversation/prompt